test/images/README.md

```diff
@@ -188,8 +178,29 @@ Finally, make sure to bump the image version used in E2E testing by modifying th
 ./build/run.sh make WHAT=test/e2e/e2e.test
 ```
 
-After all the above has been done, run the desired tests.
+After all the above has been done, run the [desired tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md) to make sure your changes work.
+
+### Promoting Images
 
+Now that you have made the changes and tested locally, you are ready to share those changes.  This is a multi step process:
+
+1. Open a pull request with your changes to the test image (the new functionality and version bump). Go through the review process and merge the pull request. See this [pull request](https://github.com/kubernetes/kubernetes/pull/99860/files) for an example.
+2. After the pull request has been approved and merged, an **automatic** postsubmit
+job will then be triggered which will build the images that were changed.  For example, if a change was
+made in `test/images/agnhost`, then the job [post-kubernetes-push-e2e-agnhost-test-images](
+https://testgrid.k8s.io/sig-testing-images#post-kubernetes-push-e2e-agnhost-test-images)
+will be triggered. The post submit job will push the images to the `gcr.io/k8s-staging-e2e-test-images` registry. You can use the image
+from the staging registry to do more testing if required. All the postsubmit jobs and their logs for all the images can be seen in
+[testgrid](https://testgrid.k8s.io/sig-testing-images) which is helpful for troubleshooting.  Note that these images are not the same as used by the e2e jobs and still need to be promoted to the final registry.
+1. The next step is to promote the image to the `k8s.gcr.io/e2e-test-images` registry by adding a line in
+[kubernetes/k8s.io](https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/images/k8s-staging-e2e-test-images/images.yaml).  See this [pull request](https://github.com/kubernetes/k8s.io/pull/1804) for an example You will need the image manifest list's digest, which can be obtained by using [manifesttool](https://github.com/estesp/manifest-tool):
+
+    ```bash
+    manifest-tool inspect --raw gcr.io/k8s-staging-e2e-test-images/${IMAGE_NAME}:${VERSION} | jq '.[0].Digest'
+    ```
```

Comment:
Should de unindented

Reply:
I needed to indent it to get the list to layout properly.  see the rendered view